docs/content/preview/yugabyte-platform/install-yugabyte-platform/install-software/default.md

```diff
@@ -121,3 +121,25 @@ Specify TLS versions via **Application config**, as shown in the following illus
 ![Application Config](/images/replicated/application-config-tls.png)
 
 The recommended TLS version is 1.2.
+
+## Set up HTTP/HTTPS proxy
+
+You can setup HTTP and HTTPS proxy via **Application config**, and select **Enable Proxy** as per the following illustration:
+
+![Enable Proxy](/images/replicated/enable-proxy.png)
+
+The following information describes details about the **Enable Proxy** setting and some considerations.
+
+- If your proxy is using default ports for protocol, then use the defaults for HTTP and HTTPS, 80 and 443 respectively, instead of 8080 and 8443 as shown in the above illustration.
```

Comment:
```suggestion
- If your proxy is using the default ports for each protocol, then set the ports for the HTTP and HTTPS proxies to the default, 80 and 443 respectively, instead of 8080 and 8443 as shown in the preceding illustration.
```